src/medCore/gui/database/medDatabaseView.cpp

```diff
@@ -11,6 +11,8 @@
 
 =========================================================================*/
 
+#include <qnamespace.h>
```

Comment:
This looks like an internal Qt header, I don't think you should include it here.